pkg/builds/parameters.go

```diff
@@ -51,7 +51,7 @@ func (p *BuildParameters) Rows() [][]string {
 		result = append(result, []string{
 			param.Type(),
 			param.Name,
-			param.Value,
+			param.Value.(string),
```

Comment:
this will raise error if `param.Value` is not `string`?

Reply:
For your information, this is [a way](https://stackoverflow.com/a/27137607) to convert `interface{}` to `string`.

Reply:
Yes. According to the [doc](https://golang.org/ref/spec#Type_assertions), if the type assertion is false, a run-time panic occurs.